statsviz.go

```diff
@@ -161,20 +160,34 @@ func (s *Server) Register(mux *http.ServeMux) {
 // intercept is a middleware that intercepts requests for plotsdef.js, which is
 // generated dynamically based on the plots configuration. Other requests are
 // forwarded as-is.
-func intercept(h http.Handler, cfg *plot.Config) http.HandlerFunc {
-	buf := bytes.Buffer{}
-	buf.WriteString("export default ")
-	enc := json.NewEncoder(&buf)
-	enc.SetIndent("", "  ")
-	if err := enc.Encode(cfg); err != nil {
-		panic("unexpected failure to encode plot definitions: " + err.Error())
+func intercept(h http.Handler, cfg *plot.Config, extraConfig map[string]any) http.HandlerFunc {
+	var plotsdefjs []byte
+	//Using parentheses helps gc
```

Comment:
`//Using parentheses helps gc`

I've never heard of that, can you point to an blog post or a benchmark or anything about that?